### PR TITLE
Add 1password and task CLIs to arkage get

### PR DESCRIPTION
**Pull request by bxffour**
## Description

Add 1password and task CLIs to arkade get

## Motivation and Context

- [ ] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
fixes: #909

## How Has This Been Tested?
Unit tests, plus test tool output

### Test tool output for task CLI

```
+ ./arkade get task --arch arm64 --os darwin --quiet
+ file /home/bxffour/.arkade/bin/task
/home/bxffour/.arkade/bin/task: Mach-O 64-bit arm64 executable, flags:<|DYLDLINK|PIE>
+ rm /home/bxffour/.arkade/bin/task
+ echo

+ ./arkade get task --arch x86_64 --os darwin --quiet
+ file /home/bxffour/.arkade/bin/task
/home/bxffour/.arkade/bin/task: Mach-O 64-bit x86_64 executable
+ rm /home/bxffour/.arkade/bin/task
+ echo

+ ./arkade get task --arch x86_64 --os linux --quiet
+ file /home/bxffour/.arkade/bin/task
/home/bxffour/.arkade/bin/task: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=PwnSGD9bn5MncgAbZgX4/aP32reofxPRUA_ZEUbQV/YnA4qqIfxJTViNfu2IpM/88pd3_evUfC0Tnyng9fL, stripped
+ rm /home/bxffour/.arkade/bin/task
+ echo

+ ./arkade get task --arch aarch64 --os linux --quiet
+ file /home/bxffour/.arkade/bin/task
/home/bxffour/.arkade/bin/task: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=rMMirTs83k5BI4yENR7b/-hhTvxNDfm42VAW1-cEG/xiYBoEzAMy3DQOtTvmte/25HORW0WRhlqwGySTkIV, stripped
+ rm /home/bxffour/.arkade/bin/task
+ echo

+ ./arkade get task --arch x86_64 --os mingw --quiet
+ file /home/bxffour/.arkade/bin/task.exe
/home/bxffour/.arkade/bin/task.exe: PE32+ executable (console) x86-64 (stripped to external PDB), for MS Windows, 6 sections
+ rm /home/bxffour/.arkade/bin/task.exe
+ echo
```

### Test tool output for 1Password CLI
```
+ ./arkade get op --arch arm64 --os darwin --quiet

The requested version of op is not available or configured in arkade for darwin/arm64

* Check if a binary is available from the project for your Operating System
* Feel free to raise an issue at https://github.com/alexellis/arkade/issues for help

Error: server returned status: 404
+ ./arkade get op --arch x86_64 --os darwin --quiet

The requested version of op is not available or configured in arkade for darwin/x86_64

* Check if a binary is available from the project for your Operating System
* Feel free to raise an issue at https://github.com/alexellis/arkade/issues for help

Error: server returned status: 404
+ ./arkade get op --arch x86_64 --os linux --quiet
+ file /home/bxffour/.arkade/bin/op
/home/bxffour/.arkade/bin/op: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=OyJkJRUAkzPLKHtdGNyx/2AoYaN9Nw3FMsui-xJ6K/LqHJCUixKdVijJ2yE4UA/ei8ed5BguIwW31R4BzY5, stripped
+ rm /home/bxffour/.arkade/bin/op
+ echo

+ ./arkade get op --arch aarch64 --os linux --quiet
+ file /home/bxffour/.arkade/bin/op
/home/bxffour/.arkade/bin/op: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=0WpxgEp-m5SrQbpPSou_/vkpInt7yNx4N5oPSGgxh/GiY6tX-2L7tFxvA_rAxg/kXLAPYUXdE2X41Yp6V6z, stripped
+ rm /home/bxffour/.arkade/bin/op
+ echo

+ ./arkade get op --arch x86_64 --os mingw --quiet
+ file /home/bxffour/.arkade/bin/op.exe
/home/bxffour/.arkade/bin/op.exe: PE32+ executable (console) x86-64 (stripped to external PDB), for MS Windows, 6 sections
+ rm /home/bxffour/.arkade/bin/op.exe
+ echo
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

**Review by alexellis — APPROVED** [review]
LGTM

File: pkg/get/tools.go [hunk 1]
```diff
@@ -3689,5 +3689,61 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
...
+					{{- end -}}
+
+					{{.Name}}_{{$os}}_{{$arch}}.{{$ext}}`,
+		})
+
+	tools = append(tools,
+		Tool{
+			Owner:       "1password",
+			Name:        "op",
+			Description: "1Password CLI enables you to automate administrative tasks and securely provision secrets across development environments.",
```
Comment (alexellis) [inline pkg/get/tools.go:3723]: Quick question about this line.

How does it render when you run `arkade get` and you see the list of CLIs?

Does it make the terminal super wide or does it get wrapped?

Comment (bxffour) [reply pkg/get/tools.go:3723]: It gets wrapped.

**Comment by alexellis** [comment]
Thank you for the PR.

We may want a shorter description for 1password

**Pull request merged**